### DISCUSSION TITLE
import of "github.com/klauspost/compress/zstd" is breaking entire package due to issue

### DISCUSSION THREAD

**Discussion by sushi4lyfe** [klauspost/compress#754] [Q&A]
...
	code.byted.org/log_market/ttlogagent_gosdk/v4/codec imports
	github.com/klauspost/compress/zstd: zip: not a valid zip file

**Comment by klauspost** [comment]
I have not idea what you are trying to say here.

**Comment by SilverRainZ** [comment]
@sushi4lyfe It is an internal issue, don't report it here please.

**Comment by wojciech-sneller** [comment]
BTW, it's worth checking the zip file using other tools (like unzip, 7zip) to find out if it's not really broken.
